### ISSUE TITLE
Proposal: add an intensity slider to the vignetting correction

### ISSUE THREAD

**Issue by bachrocks** [darktable-org/darktable#10872]
Dear darktable community

More than 2 yr darktable user and occasional Lensfun contributor speaking.
First of all, thank you very much for your great work.
My proposal: _Please add an intensity slider to the vignetting correction in the lens correction module._
Such a slider could allow between 0 and let's say 150 or 200 % vignetting correction or so.
Today, vignetting can be corrected either fully according to the Lensfun data, or not.
In my opinion, it often looks nicer to leave a certain vignette. An over correction of vignetting does always look bad (except from artistic attempts).
My request is potentially related to a previous request of a manual vignetting correction tool, https://github.com/darktable-org/darktable/issues/4578.

Regards,

Martin

**Comment by elstoc** [comment]
Also related to (or a potential duplicate of?) #1990

**Comment by MStraeten** [comment]
as a working solution you might select a different f stop in the module since the vignette effect usually degrades for smaller f stops

**Comment by elstoc** [comment]
Or duplicate the lens correction module and have the second instance just add vignetting

**Comment by bachrocks** [comment]
@MStraeten Yes I know. I suggested this already in a discussion; I've described my Lensfun data generation and openly shared my experience with the generation of a vignetting correction profile here: [Lensfun vignetting profile for the Laowa 9 mm f/2.8, and experience report](https://www.dpreview.com/forums/post/65810673). That lens exhibits strong vignetting, and I concerned myself with the topic since I wanted to have a solution that works with darktable.

@elstoc As per the duplicate to https://github.com/darktable-org/darktable/issues/1990, I agree. I had not seen that. Thank you. As per a duplication of the lens correction module, it would only help to select a different aperture for different types of correction. My proposal is to allow users to reduce (or maybe also enhance) the amount of vignetting correction.

Regards,

Martin
